app/services/forest_liana/base_getter.rb

```diff
@@ -1,5 +1,10 @@
 module ForestLiana
   class BaseGetter
+    def get_current_collection(table_name)
+      ForestLiana.apimap.find do |collection|
+        collection.name.to_s == table_name
+      end
+    end
```

Comment:
A newline after the method declaration would be more conventional in the current code.